i18n/en.json

```diff
@@ -2868,7 +2868,7 @@
   "user.settings.push_notification.allActivityOnline": "For all activity when online, away or offline",
   "user.settings.push_notification.away": "Away or offline",
   "user.settings.push_notification.disabled": "Disabled by System Administrator",
-  "user.settings.push_notification.disabled_long": "Push notifications for mobile devices have been disabled by your System Administrator.",
+  "user.settings.push_notification.disabled_long": "Push Notifications are not enabled. Contact your System Administrator.",
```

Comment:
I'm not sure about the instruction to "Contact your System Administrator".  That implies that there is something wrong which the system administrator should be made aware of.  In some situations this might be appropriate, but in our case we certainly wouldn't want users to be contacting the system administrator about this as it is not something that we will be enabling.

Reply:
That seems like a good compromise.

Reply:
PR submitted, thanks for the feedback! https://github.com/mattermost/mattermost-webapp/pull/1141